docs/scos/dev/front-end-development/202212.0/oryx/i18n/localization.md

```diff
@@ -0,0 +1,140 @@
+
+If a language resource can be resolved, the translation key is evaluated against the available translations in the resource, using all "parts" of the key. For example, if you have a translation key that contains multiple parts (e.g. `cart.increase`), you can provide a global translation for just the `increase` part or provide a translation for `cart.increase`.
+
+## Auto-conversion of translation keys
+
+If a translation key does not match any of the translations, or if the i18n feature is not installed (which is a default behavior), the translation key is auto-converted to a human readable message.
+
+For example, if you have a token that contains multiple parts (e.g. `cart.increase`), you can provide a global translation for just the `increase` part or provide a translation for `cart.increase`. This mechanism allows for a single global translation of `increase` that might affect multiple components throughout multiple components. This provides a consistent and convenient translation mechanism, while remaining flexible to add very specific localizations for some components.
+
+The table below shows a few examples to help you understand how the tokens are translated.
+
+| Token                       | Converted label |
+| --------------------------- | --------------- |
+| `cart.increase`             | Increase        |
+| `cart.add-to-cart`          | Add to cart     |
+| `cart.totals.<count>-items` | 5 items         |
+
+This mechanism allows Oryx to **not** distribute any localizations as standard package or as part of the boilerplate. The reason for this is that labels are quit opinionated and might change rapidly over time, which would cause breaking changes. For 90% of the cases the keys are fairly short and provide an OK experience.
+
+_**note:** The [Oryx labs package](https://www.npmjs.com/package/@spryker-oryx/labs) provide some localizations, mainly for demonstration reasons. More information on the labs package can be found att the [feature sets documentation](/docs/scos/dev/front-end-development/{{page.version}}/oryx/feature-setes.html)._
+
+## Install i18n package
+
+To use the i18n package, you can add the `I18nFeature` to the app and configure translation resources using the loader function:
+
+```ts
+import { appBuilder } from "@spryker-oryx/application";
+import { I18nFeature } from "@spryker-oryx/i18n";
+
+export const app = appBuilder()
+  .withFeature(
+    new I18nFeature({
+      // Here as example we are loading translations from TS modules
+      load: (localeId) => import(`../i18n/${localeId}.ts`), // <-- Required part
+    })
+  )
+  .create();
+```
+
+This gives you full flexibility to how and where to load your translation texts from. The `load` function returns a promise of a `{ default: I18nData }` type.
+
+## Translation resources
+
+In order to translate text in Oryx, you can provide translation data in a TS module. You can also integrate those in data provided by a 3rd party translation engine.

```

Comment:
What do you mean by `those` in this context?

Reply:
What about: "You can also load the text from a 3rd party translation engine."